package.json

```diff
@@ -20,6 +20,7 @@
     "postcss": "8.4.28",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.11.0",
```

Comment:
I saw this for the 3rd time in 3rd PR...from which branch you are branching-of?

Reply:
I can explain it easier tomorrow, but basically it's what we talked last week. By accident (wasn't paying attention I guess), what happened was that this branch contained everything from the other branches and when I checked git log like this : login-> videos/podcasts outline -> footer/header -> homepage  so I looked online, saw I could use rebase as a solution (your opinion on that? a friend later advised me that they tend to avoid that at work). Anyway, seemed to work fine, removed everything that was in this branch by mistake, but when I pushed to git and opened github it said that there is a conflict with the version of react-icons (there were two different ones) so i basically opted for the newer one (what is a better way to do when we have this sort of a conflict?). Anyway, accidental thing when it comes to the branch mixup (if we can call it that).